pkg/rulefmt/rulefmt_test.go

```diff
@@ -184,3 +186,120 @@ groups:
 	err1 := errs[1].(*Error).Err.node
 	require.NotEqual(t, err0, err1, "Error nodes should not be the same")
 }
+
+func TestError_Error(t *testing.T) {
+	tests := []struct {
+		name     string
+		Group    string
+		Rule     int
+		RuleName string
+		Err      WrappedError
+		want     string
+	}{
+		{
+			name:     "with alternative node provided in WrappedError",
+			Group:    "some group",
+			Rule:     1,
+			RuleName: "some rule name",
+			Err: WrappedError{
+				err: errors.New("some error"),
+				node: &yaml.Node{
+					Line:   10,
+					Column: 20,
+				},
+				nodeAlt: &yaml.Node{
+					Line:   11,
+					Column: 21,
+				},
+			},
+			want: `10:20: 11:21: group "some group", rule 1, "some rule name": some error`,
+		},
+		{
+			name:     "with node provided in WrappedError",
+			Group:    "some group",
+			Rule:     1,
+			RuleName: "some rule name",
+			Err: WrappedError{
+				err: errors.New("some error"),
+				node: &yaml.Node{
+					Line:   10,
+					Column: 20,
+				},
+			},
+			want: `10:20: group "some group", rule 1, "some rule name": some error`,
+		},
+		{
+			name:     "with only err provided in WrappedError",
+			Group:    "some group",
+			Rule:     1,
+			RuleName: "some rule name",
+			Err: WrappedError{
+				err: errors.New("some error"),
+			},
+			want: `group "some group", rule 1, "some rule name": some error`,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			err := &Error{
+				Group:    tt.Group,
+				Rule:     tt.Rule,
+				RuleName: tt.RuleName,
+				Err:      tt.Err,
+			}
+
+			got := err.Error()
+			require.Equal(t, tt.want, got)
+		})
+	}
+}
+
+func TestWrappedError_Error(t *testing.T) {
```

Comment:
I don't think we need to prefix `_Error`.

```suggestion
func TestWrappedError(t *testing.T) {
```